src/main/java/Duke.java

```diff
@@ -1,10 +1,109 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String LONG_LINE = "____________________________________________________________";
+    private static final String GREETING = "Hello! I'm Duke";
+    private static final String REQUEST = "What can I do for you?";
+    private static final String EXIT = "Bye. Hope to see you again soon!";
+    private static final String ADDED_TASK = "Got it. I've added this task:";
```

Comment:
I like that you used constants for the Strings which will make your code more readable.